ui/storybook/src/config/decorators/WithBackground.tsx

```diff
@@ -14,7 +14,7 @@
 import React from 'react';
 import { useTheme } from '@mui/material';
 import { useMemo } from 'react';
-import { DecoratorFn } from '@storybook/react';
+import { Decorator } from '@storybook/react';
```

Comment:
I missed that `DecoratorFn` is deprecated in v7 in my last PR. Fixing it while I'm in here and adding another decorator.